main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+   "fmt"
+   "log"
+   "net/http"
+   "strings"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+   fmt.Println("I got a request!")
+   fmt.Fprintf(w, "Hi there :)")
+}
+func byeHandler(w http.ResponseWriter, r *http.Request) {
+   parts := strings.Split(r.URL.Path, "/")
+   name := "human"
```

Comment:
Not sure if only humans will use this API

Reply:
This specific option is only for humans.

Reply:
You are right, sorry about that!